pkg/command/lint/command.go

```diff
@@ -81,7 +81,12 @@ func Command() *cobra.Command {
 				fmt.Fprintln(os.Stderr, "Warning: no checks enabled.")
 				return nil
 			}
-			lintCtxs, err := lintcontext.CreateContexts(args...)
+			ignorePaths, err := configresolver.GetIgnorePaths(&cfg)
+			if err != nil {
+				return err
+			}
```

Comment:
How about filtering files here instead of passing `ignorePaths`to context? I think it will be easier to follow. @dhaus67 WDYT?